### PR TITLE
Tdrd 165 publish metadata schema documentation

### PR DESCRIPTION
**Pull request by ian-hoyle**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -53,23 +58,38 @@ The [base schema](metadata-schema/baseSchema.schema.json) defines the supported
...
+        "string",
+        "null"
+      ],
+      "format": "date",
+      "alternateKeys": [
+        {
+          "tdrFileHeader": "Date of the record"
+        }
+      ],
+      "daBeforeToday": "Validates that end date is earlier than today's date"
```
Comment (kurtismash) [inline README.md:73]: Is `"Validates that end date is earlier than today's date"` present here just to make the property truthy?

File: README.md [hunk 2]
```diff
@@ -53,23 +58,38 @@ The [base schema](metadata-schema/baseSchema.schema.json) defines the supported
...
+     }
+.....  
 ```
 Definition for a ```UUID```
 * UUID - field key 
 * type - the value must be a string
 * format - the string format must be a uuid
 * tdrFileHeader - the human-readable key
```
Comment (kurtismash) [inline README.md:81]: Might work better as an annotated code block?
```
...,
"UUID": {                              // field key
    "type": "string",                  // the value must be a string
    "format": "uuid",                  // the string format must be a uuid
    "alternateKeys": [
        {
            "tdrFileHeader": "UUID"    // the human-readable key used in the TDR system
        }
    ]
},
...
```

File: README.md (hunk 1 above)
Comment (ian-hoyle) [reply README.md:73]: yes

File: README.md (hunk 2 above)
Comment (kurtismash) [reply README.md:81]: After discussion, no.

**Review by kurtismash — APPROVED** [review]
👍

**Pull request merged**